lib/ansible/modules/cloud/vmware/vmware_guest_vnc.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Armin Ranjbar Daemi randjbar@gmail.com
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_guest_vnc
+
+short_description: Manages VNC remote display on virtual machines in vCenter
+
+description:
+  - Enables and disables VNC remote display on virtual machine.
+
+version_added: '2.6'
+
+author:
+  - Armin Ranjbar Daemi (@rmin) <randjbar@gmail.com>
+
+requirements:
+  - python >= 2.6
+  - PyVmomi
+
+options:
+  hostname:
+    description:
+      - vSphere or VCenter hostname.
+    required: true
+  username:
```

Comment:
Username, password, port and validate_certs are already included in vmware.documentation

Reply:
Ok, removing them.